test/register-login.spec.js

```diff
@@ -0,0 +1,126 @@
+const {
+    expect
+} = require("chai")
+const {
+    By,
+    Builder,
+    until,
+    elementLocated,
+    error
+} = require("selenium-webdriver");
+
+describe('Test performs the testing of the login and sign up forms', () => {
+    let driver;
+    let testDataRandomizer = () => {
+        return Math.random().toString(36).replace(/[^a-z]/g, '').substr(0, 11);
+    };
+    let testNumbersRandomizer = () => {
+        return (Math.floor(Math.random() * (98765432120 - 100)) + 10000);
```

Comment:
I recommend the usage of specially designed libraries for the purpose of creating random strings: `randomstaring`, `chance`, and so on.